branding/pdf/entities.adoc

```diff
@@ -12,6 +12,7 @@
 :arm64: ARM64
 :centos: CentOS
 :oracle: Oracle Linux
+:amazonlinux: Amazon Linux
```

Comment:
Perhaps make this `:amazon:` rather than the whole thing?

Reply:
TBH I'd have use `:oraclelinux:` rather than `Oracle`, in case we want to sue the company names as entities as well. But as you prefer. Should I change it?

Reply:
Changed.